bot.js

```diff
@@ -13,38 +14,30 @@ if (process.env.NODE_ENV === 'production') {
 }
 
 bot.on('message', function (msg) {
-  if (storage == null) {
-    // bot.sendMessage(msg.chat.id, "Что это было, " + msg.from.first_name + "?")
+  const chatID = msg.chat.id
+  if (storage.answer == null) {
+    // bot.sendMessage(chatID, 'The game is not started! Start the game by typing /play')
     return
   }
-  const wordInfo = storage['currentWordInfo']
-  const rightWord = wordInfo['rightWord']
-  const isRightAnwer = (msg.text === rightWord.ar)
-  if (isRightAnwer) {
-    bot.sendMessage(msg.chat.id, 'Красавчик!')
+
+  if (msg.text === storage.answer) {
+    bot.sendMessage(chatID, 'Well done!')
   } else {
-    bot.sendMessage(msg.chat.id, 'Упс, правильно было: ' + rightWord.ar)
+    bot.sendMessage(chatID, 'Oops, the right answer is: ' + storage.answer)
   }
-  storage = null
+  delete storage.answer
 })
 
 bot.onText(/\/start/, (msg) => {
   bot.sendMessage(msg.chat.id, 'Welcome ' + msg.from.first_name)
 })
 
 bot.onText(/\/play/, (msg) => {
-  const wordInfo = quiz.randomWordInfo()
-  const rightWord = wordInfo['rightWord']
-  const question = quiz.questionText(rightWord)
-
-  const variants = utils.shuffleArray(wordInfo['words'].map(word => {
-    return word.ar
-  }))
-
-  const opts = { 'reply_markup': { 'keyboard': [variants], 'one_time_keyboard': true } }
-  storage = { 'currentWordInfo': wordInfo }
+  const question = quiz.generateQuestion(words)
+  storage.answer = question.answer
```

Comment:
Why don't you use `storage.chatID` as you suggested?

Reply:
It's seems, he did not intent to use it in this PR, i think it would a particular PR for that.

Reply:
Yeah. That makes a sense.